shark_turbine/kernel/wave/expansion.py

```diff
@@ -383,14 +427,17 @@ def get_dim_scaling(
                     "Tile size must be divisible by wave count and vector size"
                 )
             dim_scaling[constraint.dim] = tile_size // wave_count // vector_size
-    return dim_scaling
+            dim_tile_size[constraint.dim] = index_symbol(f"$TILE_SIZE_{constraint.dim}")
+            tile_size_mapping[dim_tile_size[constraint.dim]] = vector_size
+    return (dim_scaling, dim_tile_size, tile_size_mapping)
```

Comment:
I am not sure I understand the name of `tile_size_mapping`. This maps a tile_size symbol to the corresponding vector size. Does it make sense to name it `dim_vector_size` then? Maybe there is another good name that I am not seeing right now

Reply:
sure let me rename this.

Reply:
I have removed this from the code.